dbt/include/impala/macros/apply_grants.sql

```diff
@@ -0,0 +1,28 @@
+{#-- Assume grants copy over --#}
+{% macro impala__copy_grants() %}
+    {{ return(True) }}
+{% endmacro %}
+
+
+{%- macro impala__get_grant_sql(relation, privilege, grantees) -%}
+    grant {{ privilege }} on {{ relation }} to {{ adapter.quote(grantees[0]) }}
```

Comment:
Is this not automatically taken care by https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/include/global_project/macros/adapters/apply_grants.sql#L73. What error are we receiving here.

Reply:
impala doesn't support multiple grantees for a single grant statement: https://impala.apache.org/docs/build/html/topics/impala_grant.html